cost-model/src/transaction_cost.rs

```diff
@@ -67,11 +67,80 @@ impl TransactionCost {
     }
 
     pub fn sum(&self) -> u64 {
-        self.signature_cost
-            .saturating_add(self.write_lock_cost)
-            .saturating_add(self.data_bytes_cost)
-            .saturating_add(self.builtins_execution_cost)
-            .saturating_add(self.bpf_execution_cost)
-            .saturating_add(self.loaded_accounts_data_size_cost)
+        if self.is_simple_vote {
+            self.signature_cost
+                .saturating_add(self.write_lock_cost)
+                .saturating_add(self.data_bytes_cost)
+                .saturating_add(self.builtins_execution_cost)
+        } else {
+            self.signature_cost
+                .saturating_add(self.write_lock_cost)
+                .saturating_add(self.data_bytes_cost)
+                .saturating_add(self.builtins_execution_cost)
+                .saturating_add(self.bpf_execution_cost)
+                .saturating_add(self.loaded_accounts_data_size_cost)
+        }
+    }
+}
```

Comment:
a little dereplicode nit

```suggestion

        self.signature_cost
            .saturating_add(self.write_lock_cost)
            .saturating_add(self.data_bytes_cost)
            .saturating_add(self.builtins_execution_cost)
        if !self.is_simple_vote {
            self.signature_cost
                .saturating_add(self.bpf_execution_cost)
                .saturating_add(self.loaded_accounts_data_size_cost)
        }
}
```

though i guess we really want this change for _all_ native instructions, not just votes?

Reply:
Yea, Ashwin also pointed it https://github.com/solana-labs/solana/pull/33235#discussion_r1324808267 I did it verbosely so can remove "vote path" safer if #33230 lands.


> though i guess we really want this change for all native instructions, not just votes?

The "cost" here is sum-ed up for transaction - all native instructions has const cost, all roll up into transaction's cost. Making a special case for Vote *transaction*.